src/lib/backend/index.js

```diff
@@ -0,0 +1,15 @@
+import gsheetsFn from "@/promisetracker/lib/gsheets";
+import serverFn from "@/promisetracker/lib/server";
+
+function backend(siteSlug) {
+  const server = serverFn(siteSlug);
+  const backendChoice = server.env("BACKEND");
+  switch (backendChoice?.toUpperCase()) {
+    case "GSHEET":
+      return gsheetsFn(server);
+    default:
```

Comment:
@kilemensi  Shouldn't we also have the JSON case here since the functionality is already there? Or will it also require some updating?

Reply:
We need JSON functionality to support URL @esirK ... we'll only be caching files in public folder.